ProjectLighthouse/DigestUtils.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LBPUnion.ProjectLighthouse
+{
+    public static class DigestUtils
```

Comment:
This should probably be called `DigestHelper` and be put in the `Helpers` directory. Or you could also put it in `HashHelper`. Up to you.